R/utilities.R

```diff
@@ -1376,7 +1380,7 @@ collate_results = function(sample_table,
       meta_data = get_gambl_metadata(seq_type_filter = seq_type_filter)
     }
 
-    full_table = left_join(sample_table, meta_data)
+    full_table = left_join(meta_data, sample_table)
```

Comment:
Out of interest, why was this change made?

Reply:
I reverted a change that I made in [this](https://github.com/morinlab/GAMBLR/pull/133/commits/213e9f8f0e3721fefbf3e0de9164173f96d0adfd) commit, from when I first started looking into this function last week.

Reply:
Then i have the same question but in regards to that commit instead. Why was it changed?

Reply:
I thought it should be the other way around (i.e sample_table , meta_data) but after some thinking and testing, it was correct from the beginning (i.e meta_data, sample_table).